third_party/2and3/redis/client.pyi

```diff
@@ -242,14 +242,14 @@ class Redis(object):
         self, name: _Key, mapping: Mapping[_Key, _Str], nx: bool = ..., xx: bool = ..., ch: bool = ..., incr: bool = ...
     ) -> int: ...
     def zcard(self, name): ...
-    def zcount(self, name, min, max): ...
+    def zcount(self, name: _Key, min: _Str, max: _Str): ...
```

Comment:
`min` and `max` can also be a `float` value. It returns an `int`.

Reply:
See the definition of `_Str`: `_Str = Union[bytes, float, int, Text]`. We can change the name of the variable to `_StrOrNumber` if you'd like?

Reply:
Sorry for missing that. We should maybe change it (maybe just to `_Value`?) at some point, but I believe it is out of scope for this PR.

Reply:
I'll do it in another PR!